rocAL/rocAL_pybind/amd/rocal/fn.py

```diff
@@ -255,339 +473,536 @@ def resize(*inputs, bytes_per_sample_hint=0, image_type=0, interp_type=1, mag_fi
+
+def color_twist(*inputs, brightness=1.0, contrast=1.0, hue=0.0,
+                saturation=1.0, rocal_tensor_output_layout=types.NHWC, rocal_tensor_output_datatype=types.UINT8):
+    """
+    inputs - the input image passed to the augmentation
+
+    brightness (float, optional, default = 1.0) - brightness multiplier. Values >= 0 are accepted. For example: 0 - black image, 1 - no change, 2 - increase brightness twice
+
+    contrast (float, optional, default = 1.0) - contrast multiplier used for the augmentation. Values >= 0 are accepted. For example: 0 - gray image, 1 - no change, 2 - increase contrast twice
+    
+    hue (float, optional, default = 0.0) - hue change in degrees
+
+    saturation (float, optional, default = 1.0) - The saturation change factor. Values must be non-negative. Example values: 0 - Completely desaturated image, 1 - No change to image's saturation.
+
+    rocal_tensor_output_layout (int, optional, default = types.NHWC) - tensor layout for the augmentation output
+
+    rocal_tensor_output_datatype (int, optional, default = types.UINT8) - tensor dtype for the augmentation output
+    """
+    brightness = b.createFloatParameter(brightness) if isinstance(brightness, float) else brightness
+    contrast = b.createFloatParameter(contrast) if isinstance(contrast, float) else contrast
+    hue = b.createFloatParameter(hue) if isinstance(hue, float) else hue
+    saturation = b.createFloatParameter(saturation) if isinstance(saturation, float) else saturation
+
+    # pybind call arguments
+    kwargs_pybind = {"input_image": inputs[0], "is_output": False, "p_alpha": brightness, "p_beta": contrast,
+                     "p_hue": hue, "p_sat": saturation, "rocal_tensor_output_layout": rocal_tensor_output_layout, "rocal_tensor_output_datatype": rocal_tensor_output_datatype}
+    color_twist_image = b.colorTwist(Pipeline._current_pipeline._handle, *(kwargs_pybind.values()))
     return (color_twist_image)
 
-def uniform(*inputs,rng_range=[-1, 1], device=None):
-    output_param = b.CreateFloatUniformRand(rng_range[0], rng_range[1])
+
+def uniform(*inputs, rng_range=[-1, 1]):
+    """
+    inputs - the input image passed to the augmentation
+ 
+    rng_range (list of ints, optional, default = [-1, 1]) - uniform distribution used for random number generation
+    """
+    output_param = b.createFloatUniformRand(rng_range[0], rng_range[1])
     return output_param
 
-def random_bbox_crop(*inputs,all_boxes_above_threshold=True, allow_no_crop=True, aspect_ratio=None, bbox_layout="", bytes_per_sample_hint=0,
-                crop_shape=None, input_shape=None, ltrb=True, num_attempts=1 ,scaling=None, preserve=False, seed=1, shape_layout="",
-                threshold_type="iou", thresholds=None, total_num_attempts=0, device=None, labels=None):
+
+def random_bbox_crop(*inputs, all_boxes_above_threshold=True, allow_no_crop=True, aspect_ratio=None,
```

Comment:
what happened to the other parameters line ltrb, threshold_type etc. We need to keep all relevant parameters name same as dali for easy porting of dali to rocal

Reply:
I retained all the parameters that are passed to C++ but some of the other parameters were present only to make API parameters similar to DALI but were not passed to C++. I removed those unused parameters from the function

Reply:
As discussed in last call, added most of the parameters from DALI even if its unused and will be adding docstrings for those params as "Parameter unused in augmentation" in the doxygen PR

Reply:
We need ltrb and threashold type. Does the low level api assume it is ltrb?